.github/workflows/openmetadata-ingestion-core-version-comment.yml

```diff
@@ -0,0 +1,40 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+name: OpenMetadata Ingestion Core Version Check Comment
+
+# read-write repo token
+# access to secrets
+on:
+  pull_request_target:
+    branches: [ main ]
+    types: [ opened ]
+    paths:
+    - 'catalog-rest-service/src/main/resources/json/**'
```

Comment:
If I remember correctly from previous tests the filter does not quite work for the pull_request_target. We might need to check https://github.com/open-metadata/OpenMetadata/issues/2185 and use https://github.com/dorny/paths-filter to validate the paths on the checked out code!

Reply:
As per documentation, this should work with `pull_request_target`. 
Ref - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Then I cannot remember why I started looking into that action 😂  I must be mixing something here. LGTM!